File: terminal/subte.py
#!/usr/bin/env python
import requests
from BeautifulSoup import BeautifulSoup

text = requests.get('http://www.metrovias.com.ar').text
b = BeautifulSoup(text)
res = ""
for line in "ABCDEH":
    text = b.find(id=("status-line-%s" % line)).text
    if text == 'Normal' or 'habitual' in text:
        res += "\033[42m"
    elif "emora" in text or "obras" in text:
        res += "\033[43m"
    else:
        res += "\033[41m"
    res += "\033[1m|" + line + "|\033[0m "
print res

File: firefox_addon/popup/subte.js
$.ajax({
    url: "http://www.metrovias.com.ar",
    type: 'GET',
    success: function(data){
        var data = $(data);
        var lines = ["A", "B", "C", "D", "E", "H"];
        for (var i = 0; i < lines.length; ++i) {
            var text = data.find("span#status-line-" + lines[i]).text();
            var selector = $("div#" + lines[i] + " > span.status");
            if (text == "Normal" || text.indexOf("habitual")>-1){
                selector.text("✓");
                selector.addClass('ok');
            } else if (text.indexOf("emora")>-1 || text.indexOf("obras")>-1){
                selector.text("⚠");
                selector.addClass('warn');

            } else {
                selector.text("❌");
                selector.addClass('bad');
            }
        }
    }
});
